types/api/Sources.d.ts

```diff
@@ -715,6 +715,46 @@ declare module '@stripe/stripe-js' {
 
       statement_descriptor?: string;
     }
+
+    interface SofortCreate {
```

Comment:
It seems like it might make more sense to move these new type definitions into `namespace SourceCreateParams` below, and remove `Create` from their names. The new references could then use types like `SourceCreateParams.Sofort`, which would seem to be more consistent with existing usage, e.g. `SourceCreateParams.Mandate`: https://github.com/stripe/stripe-js/blob/8d17900d282e3f68d54b1a31bb822ebd43022acc/types/api/Sources.d.ts#L789